services/web/server/src/simcore_service_webserver/projects/exceptions.py

```diff
@@ -118,6 +119,10 @@ class ProjectNodeResourcesInsufficientRightsError(BaseProjectError):
     ...
 
 
+class ProjectNodeRequiredInputsNotSetError(BaseProjectError):
+    ...

```

Comment:
NOTE that the user will receive a message like "Unexpected error occured in projects submodule" sice this exception has no `msg_template`